src/components/Filters/components/Date/FastDayPicker.js

```diff
@@ -197,8 +200,8 @@ class FastDayPicker extends Component {
             onChange={this.handleDateInputChange}
             onKeyDown={this.handleKeyDown}
             format="l"
-            overlayComponent={this.getOverlayComponent}
-            showOverlay
+            // overlaycomponent={this.getOverlayComponent}
```

Comment:
What's wrong with overlays?

Reply:
it writes an error that the property doesnot exist